src/CarouselIndicators.js

```diff
@@ -13,7 +13,7 @@ const CarouselIndicators = (props) => {
     ), cssModule);
     return (
       <li
-        key={`${item.key || item.src}${item.caption}${item.altText}`}
+        key={`${Object.values(item).join('')}`}
```

Comment:
It would still be good to explicitly allow the key to be the key specified if specified. In the case where other values change but the user wants to ensure its the same instance of the component, an explicit key would be the only way to ensure that.

Reply:
100% Agree. What about something like this?

```suggestion
        key={`${item.key || Object.values(item).join('')}`}
```

Reply:
Yes, exactly.